### PR TITLE
LPS-122912

### PR DESCRIPTION
**Pull request by carolmariaabb**
**Related Issue:** https://issues.liferay.com/browse/LPS-122912

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_10_2/util/DDMFormFieldUpgradeProcessUtil.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONFactoryUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.kernel.util.Validator;
+
+/**
+ * @author Carolina Barbosa
+ */
+public class DDMFormFieldUpgradeProcessUtil {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_10_2/util/DDMFormFieldUpgradeProcessUtil.java:26]: Nice job, @carolmariaabb!

**Review by natocesarrego — APPROVED** [review]

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**